packages/jest-expo/src/enzyme/setupEnzyme.native.js

```diff
@@ -0,0 +1,45 @@
+import 'react-native';
+import 'jest-enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import Enzyme from 'enzyme';
+import serializer from './serializer';
+
+/**
+ * Set up DOM in node.js environment for Enzyme to mount to
+ */
+const { JSDOM } = require('jsdom');
+
+const jsdom = new JSDOM('<!doctype html><html><body></body></html>');
+const { window } = jsdom;
+
+function copyProps(src, target) {
+  Object.defineProperties(target, {
+    ...Object.getOwnPropertyDescriptors(src),
+    ...Object.getOwnPropertyDescriptors(target),
+  });
+}
+
+global.window = window;
```

Comment:
note that this is discouraged by jsdom itself (https://github.com/jsdom/jsdom/wiki/Don't-stuff-jsdom-globals-onto-the-Node-global). Weird that this is still the way enzyme wants to do things.

Anyways, since this is opt-in I guess it's ok, even though it uses ugly hacks 🙂